bodhi/tests/server/base.py

```diff
@@ -32,6 +34,21 @@
 DB_NAME = None
 
 
+FAITOUT = 'http://209.132.184.152/faitout/'
+# The BUILD_ID environment variable is set by Jenkins and allows us to detect if
+# we are running the tests in jenkins or not
+# https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-below
+if os.environ.get('BUILD_ID'):
+    try:
+        req = requests.get('%s/new' % FAITOUT)
```

Comment:
You might want to add a timeout kwarg here since the default is to never time out and that might cause the tests to hang for an indefinite amount of time. Not a blocker since it was already like that, but just so you know.